lib/data_update_scripts/20220429142653_remove_mailchimp_sustaining_members_newsletter.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class RemoveMailchimpSustainingMembersNewsletter
+    def run
+      Settings::General.where(var: "mailchimp_sustaining_members_id").delete_all
```

Comment:
I think this might need to be `#destroy_all` to trigger the after_commit hook on `Settings::Base` to `clear_cache`?